Prog/mt_vern_profile_data.sas

```diff
@@ -356,15 +360,17 @@ proc transpose data=price_tab_&geosuf out=price_&geosuf prefix=mprice;
     id saleyear;
     var saleprice_median;
 run;
-	
+
+%if &geo = ward2012 %then %do;	
 data compile_mvt_tabs_wd12;
 	merge compile_mvt_tabs_wd12 price_wd12 (keep= &geo &price_vars);
-run;
-
+	by &geo;
+run; %end;
+%else %if &geo = city %then %do;
 data compile_mvt_tabs_city;
```

Comment:
this should not overwrite the existing temp file. put in a new name. (best practice).  also i'm getting missing values in the acs, tanf, data etc. on the compile_mvt_tabs_city file.

Reply:
should the project_&geosuf files get set in this statement too?

Reply:
it looks like there are several places where you are overwriting data sets -please review the whole program for this and correct.